app/views/govuk_component/option_select.raw.html.erb

```diff
@@ -1,25 +1,23 @@
-<div class="govuk-option-select" <% if local_assigns.include?(:closed_on_load) && closed_on_load %>data-closed-on-load="true"<% end %>>
+<% title_id = "option-select-title-#{title.parameterize}" %>
+<div class="govuk-option-select"
+  <% if local_assigns.include?(:closed_on_load) && closed_on_load %>data-closed-on-load="true"<% end %>
+  <% if local_assigns.include?(:aria_controls_id) %>data-input-aria-controls="<%= aria_controls_id %>"<% end %>
+>
   <div class="container-head js-container-head">
-    <div class='option-select-label'><%= title %></div>
+    <div class="option-select-label" id="<%= title_id %>">
+      <%= title %>
+    </div>
   </div>
-  <div class='options-container' id='<%= options_container_id %>'>
-    <div class='js-auto-height-inner'>
+  <div role="group" aria-labelledby="<%= title_id %>" class="options-container" id="<%= options_container_id %>">
```

Comment:
Do we need to check this change in different A.T?

Reply:
Yes. I've only been able to test with VoiceOver on Mac and iOS so far.